src/request-data/resource.js

```diff
@@ -62,14 +68,24 @@ const _container = Symbol('container');
 const _abortController = Symbol('abortController');
 class Resource extends BaseResource {
   constructor(container, name) {
-    const store = shallowReactive({ data: null, awaitingResponse: false });
+    const store = shallowReactive({
+      data: null,
+      awaitingResponse: false,
+      setAt: null,
+      patchedAt: null
+    });
     super(name, store);
     this[_container] = container;
     this[_abortController] = null;
   }
 
   get data() { return this[_store].data; }
-  set data(value) { this[_store].data = value; }
+
+  set data(value) {
+    this[_store].data = value;
+    this[_store].setAt = new Date();
```

Comment:
`setAt` is only set: it's never cleared. I thought about setting it to `null` whenever `data` was set to `null`, but I ended up thinking that that wasn't necessary. I think it's simpler if `setAt` is always the timestamp of when `data` was last set, whether to `null` or anything else.

I took a similar strategy for `patchedAt`. I thought about setting `patchedAt` to `null` whenever `data` was set (either to `null` or to anything). But again, it ended up feeling unnecessary and just extra logic.

I'm happy to take another approach, this just seemed like the simplest approach.